src/core/textrenderer/qgstextrenderer.h

```diff
@@ -239,6 +239,40 @@ class CORE_EXPORT QgsTextRenderer
      */
     static constexpr double FONT_WORKAROUND_SCALE = 10;
 
+    /**
+     * Returns the number of lines needed to fully display the text
+     * \param context renderer context
+     * \param text text to be used
+     * \param space space available to display the text
+     * \param format text formatting
+     *
+     * \since QGIS 3.20
+     */
+    static bool requiresWrapping( QgsRenderContext &context, const QString &text, double space, const QgsTextFormat &format );
```

Comment:
const context? also can we have context as last argument?

Reply:
As this came from qgslayouttable I didn't want to reorder inputs, though the context is mandatory to render things in order to asses the space taken. 

Though yes consting it shouldn't be an issue.